internal/cli/bor_fingerprint.go

```diff
@@ -101,9 +105,11 @@ func (c *FingerprintCommand) checkFio() error {
 // Run the IOPS benchmark for the node
 func (c *FingerprintCommand) benchmark() error {
 	var b []byte
+
 	err := c.checkFio()
+
 	if err != nil {
-		return nil
+		return err
```

Comment:
@0xsharma Could you confirm if previous `return nil` was a typo, which is supposed to be `return err`?

Reply:
@0xsharma Could you help with the question above?

Reply:
@cffls @JekaMas 
It is supposed to be `return nil ` since it prints that "Fio is not installed"(in func checkFio) and does not perform benchmark if checkFio returns err. Also, we just want to gently exit the fingerprint test asking the user to install Fio and not output any error.

Reply:
I see. Thank you!

@cffls I believe with this revert we can merge the pr.

Reply:
Thanks @0xsharma @JekaMas ! I will revert this change and add a comment in the code.